### PR TITLE
Mergeback 1.4.1 to develop

### PR DESCRIPTION
**Pull request by yunchu**
### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [ ] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: README.md
Comment (goodsong81) [inline README.md:1]: "latest" is the right link.

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 ---
 
-[Key Features](#key-features) •
-[Installation](https://openvinotoolkit.github.io/training_extensions/latest/guide/get_started/installation.html) •
-[Documentation](https://openvinotoolkit.github.io/training_extensions/latest/index.html) •
+[Key Features](#key-features)
+[Installation](https://openvinotoolkit.github.io/training_extensions/1.4.1/guide/get_started/installation.html)
+[Documentation](https://openvinotoolkit.github.io/training_extensions/1.4.1/index.html)
```
Comment (goodsong81) [inline README.md]: Was the "•" removed intentionally?

File: docs/source/guide/explanation/additional_features/auto_configuration.rst
Comment (goodsong81) [inline docs/source/guide/explanation/additional_features/auto_configuration.rst:1]: "latest" here, too?

**Review by goodsong81 — CHANGES_REQUESTED** [review]
Why don't we merge only commits after the latest merge-back?
Anyway, "latest" should be applied to docs.

File: README.md (hunk 1 above)
Comment (yunchu) [reply README.md]: oh..it's my fault. those are the separators. will revert it.

**Review by goodsong81 — APPROVED** [review]

**Review by sungmanc — APPROVED** [review]
LGTM

**Review by eunwoosh — APPROVED** [review]

**Pull request merged**